package/MDAnalysis/core/AtomGroup.py

```diff
@@ -4671,12 +4865,14 @@ def trajectory(self, value):
         del self._trajectory  # guarantees that files are closed (?)
         self._trajectory = value
 
+    @deprecate(message=_SIXTEEN_DEPRECATION)
```

Comment:
What does it do right now? It doesn't exist as a concept in 363, so I removed it, but is it necessary given that AtomGroups no longer point to Atoms, and that Atoms no longer exist without a Universe?